docs/core/run-time-config/compilation.md

```diff
@@ -8,11 +8,118 @@ ms.topic: reference
 
 ## Tiered compilation
 
-- Configures whether the JIT compiler uses [tiered compilation](../whats-new/dotnet-core-3-0.md#tiered-compilation).
+- Configures whether the just-in-time (JIT) compiler uses [tiered compilation](../whats-new/dotnet-core-3-0.md#tiered-compilation). Tiered compilation includes two distinct behaviors: [quick JIT](#quick-jit) and the use of [ReadyToRun](../whats-new/dotnet-core-3-0.md#readytorun-images) (pre-compiled) code.
 - In .NET Core 3.0 and later, tiered compilation is enabled by default.
 - In .NET Core 2.1 and 2.2, tiered compilation is disabled by default.
+- For more information, see the [Tiered compilation guide](https://github.com/dotnet/runtime/blob/master/docs/design/features/tiered-compilation-guide.md).
```

Comment:
not related to this PR, but this seems like something we should add to our docs, right?

Reply:
Created https://github.com/dotnet/docs/issues/16805